tests/modules/programs/lsd/example-settings.nix

```diff
@@ -0,0 +1,49 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.lsd = {
+      enable = true;
+      enableAliases = false;
+      settings = {
+        date = "relative";
+        blocks = [ "date" "size" "name" ];
+        color.theme = "custom";
```

Comment:
If you add the automatic assignment of the `color.theme` option then can remove this line.
```suggestion
```